src/pages/Dashboard/components/DashboardHeader/DashboardHeaderTop.tsx

```diff
@@ -113,7 +113,7 @@ export const DashboardHeaderTop = memo(() => {
       >
         <IconButton isRound icon={moreIcon} aria-label='Settings' onClick={onOpen} />
       </Flex>
-      {isMobile ? (
+      {!isMobile ? (
```

Comment:
Revert me before opening or 🔫

Reply:
Done in [7c200bd](https://github.com/shapeshift/web/pull/8760/commits/7c200bd5a4d18cccddd6a13aeaaa1fed3e09138d)